Makefile

```
@@ -28,6 +29,12 @@ helm-lint: helm-lint-$(notdir $(CHART_DIR)/*)
 helm-lint-%:
 	$(HELM) lint $(CHART_DIR)/$*
 
+deploy-cert-manager:
+	kubectl create namespace cert-manager
+	$(HELM) install --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
```

```suggestion
	$(HELM) upgrade -i --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
```
This will do upgrades and installs

---

Done